service/JobRoleService.ts

```diff
@@ -0,0 +1,16 @@
+import { JobRole } from "../model/JobRole";
+
+const axios = require('axios');
+
+module.exports.getJobRoles = async function()
+{
+    try
+    {
+        const response = await axios.get('http://localhost:8080/api/job-roles')
```

Comment:
Could we type this response?

```suggestion
const response: axios.get('http://localhost:8080/api/job-roles')
```

Reply:
`const res:AxiosResponse = await axios.get('http://localhost:8080/api/job-roles')`
Is this what you mean by typing the response?